scapy/layers/bluetooth.py

```diff
@@ -18,6 +20,26 @@
 from scapy.supersocket import SuperSocket
 from scapy.sendrecv import sndrcv
 from scapy.data import MTU
+from scapy.consts import WINDOWS
+from scapy.error import warning, log_loading
+
+BT_AVAILABLE = True
+PYBLUEZ = False
+
+if WINDOWS:
+    from scapy.arch.windows import _test_windows_bluetooth
+    if not _test_windows_bluetooth():
+        log_loading.info("Bluetooth is not available on this computer")
+        BT_AVAILABLE = False
+
+try:
+    import bluetooth
```

Comment:
I'm not sure to understand this: will we use the `bluetooth` module under other OSes than Windows?

Reply:
the `bluetooth` module, or PyBluez, will be used if available. They use quite a lot of tricks to access the bluetooth sockets on Windows.
It's compulsory for Windows, optional for the other OSes.